backend/Dockerfile

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 # 1. Build api server application
-FROM golang:1.20.4-buster as builder
+FROM golang:1.21.7-bookworm as builder
```

Comment:
debian:stable is now based on the bookworm image. https://www.debian.org/releases/

Reply:
No updating dockerfile in this PR anymore